doc/panes/editor.rst

```diff
@@ -27,35 +27,273 @@ The Editor pane consists of the following areas:
+   :alt: Spyder's preferences dialog, showing checking the autoformat files on save setting
+
+============
+Running code
+============
+
+As previously mentioned, the Editor is the place where you easily manage your work.
+It allows you to create a new file or open an existing one.
+It also lets you run an entire file as well as certain lines or selections.
+As your code is running,
+
+* The :doc:`ipythonconsole` will display the results and errors when applicable
+* The :doc:`variableexplorer` allows you to browse and interact with the objects generated when running your code.
+* The :doc:`plots` pane shows the figures and images created during your code execution.
+* The :doc:`profiler` helps you optimize your code by determining the run time and number of calls for every function and method used in a file. It also allows you to save and compare your results between runs.
+
+--------
+Run file
+--------
+
+You can run any source file in the Editor by simply using the Run file button in Spyder’s toolbar or by pressing the F5 key.
+You can also run your code by selecting  :menuselection:`Run --> Run` menu item.
+
+--------
+Run line
+--------
+
+You can execute just a single line of your source code where the cursor currently resides by selecting the :guilabel:`Run selection or current line` option from the  :menuselection:`Run` menu or Spyder’s toolbar.
+Alternatively, you can use the F9 key to run any line.
+After executing your selected line of code, the cursor automatically advances to the next line.
+This helps you run your code in sequential order with a single line step.
+
+-----------------
+Other run options
+-----------------
+
+In addition to executing your entire source file at once or just a single line, the Editor allows you to run your code in other options.
+
+^^^^^^^^^^^^^^
+Multiple lines
+^^^^^^^^^^^^^^
+
+You can execute multiple lines from within the editor by highlighting these lines and using the :guilabel:`Run selection or current line` option from the  :menuselection:`Run` menu or Spyder’s toolbar.
+Alternatively, you can use the F9 key to run any line.
+After executing a selection of code, use the :guilabel:`Re-Run last script` command from the :menuselection:`Run` menu to execute the same selection again.
+The main difference between running a selection of lines and an entire file is that in the former all lines are inserted directly into the :doc:`ipythonconsole` whereas in the latter only the results are shown in the console.
+
+^^^^^^^^^^
+Code cells
+^^^^^^^^^^
+
+To run a cell, press :kbd:`Ctrl-Return` (while your cursor is focused on it) or use the :guilabel:`Run current cell` button in Spyder’s toolbar.
+
+^^^^^^^^^^^^^^^^^
+Run configuration
+^^^^^^^^^^^^^^^^^
+
+You can configure the :guilabel:`Run per file` settings to control where and how this file runs in the :guilabel:`Configuration per file…` dialog under the :menuselection:`Run` menu or pressing :kbd:`Ctrl+F6`.
```

Comment:
Are you using `+` or `-` for shortcut combinations? I think this is the first one that doesn't use `-`

```suggestion
You can configure the :guilabel:`Run per file` settings to control where and how this file runs in the :guilabel:`Configuration per file…` dialog under the :menuselection:`Run` menu or pressing :kbd:`Ctrl-F6`.
```